src/docbuilder/rustwide_builder.rs

```diff
@@ -61,6 +52,9 @@ static ESSENTIAL_FILES_UNVERSIONED: &[&str] = &[
     "SourceSerifPro-It.ttf.woff",
 ];
 
+static DUMMY_CRATE_NAME: &str = "acme-client";
```

Comment:
Uh, could we choose a less... likely to exist dummy crate? Maybe use a rustc-dep-of-std dep which is ~guaranteed empty?

Looking at the code maybe we just want a `Limit::default()`

Reply:
> Looking at the code maybe we just want a Limit::default()

The dummy crate is not used just for limits, it's actually built to then capture the essential files after the toolchain is updated (those constants are used a few lines below to create the `Crate` struct.

> Uh, could we choose a less... likely to exist dummy crate? Maybe use a rustc-dep-of-std dep which is ~guaranteed empty?

I just ported the crate name from the old code :sweat_smile: 

The crate actually exists but in version `0.0.0` is empty, so we're basically building an empty crate to then extract the essential files.